lib/human_player.rb

```diff
@@ -0,0 +1,114 @@
+=begin
+Winning patterns
+  #horizontal wins
+  [0][0], [0][1], [0][2]
+  [1][0], [1][1], [1][2]
+  [2][0], [2][1], [2][2]
+
+  #vertical wins
+  [0][0], [1][0], [2][0]
+  [0][1], [1][1], [2][1]
+  [0][2], [1][2], [2][2]
+
+  #diagonal wins
+  [0][0], [1][1], [2][2]
+  [0][2], [1][1], [2][0]
+=end
```

Comment:
These comments are great! They will serve as good examples for when you practice TDD on the `#win` method.